src/libraries/System.Security.Cryptography.Cose/tests/CoseTestHelpers.cs

```diff
@@ -0,0 +1,136 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Microsoft.IdentityModel.Tokens;
+using Test.Cryptography;
+
+namespace System.Security.Cryptography.Cose.Tests
+{
+    public class CoseTestHelpers
+    {
+        internal static readonly byte[] s_SampleContent = Encoding.UTF8.GetBytes("This is the content.");
+        internal const string ContentTypeDummyValue = "application/cose; cose-type=\"cose-sign1\"";
+        internal const string SHA256 = KnownHashAlgorithms.SHA256;
+        internal const string SHA384 = KnownHashAlgorithms.SHA384;
+        internal const string SHA512 = KnownHashAlgorithms.SHA512;
+
+        public enum ECDsaAlgorithm
+        {
+            ES256 = KnownCoseAlgorithms.ES256,
```

Comment:
When I commented above about the test wanting to have it's own definition of the value, I didn't realize that this enum was different than what the product code used.

That said, it should redefine it as -7 here, instead of sharing the const with the source project :smile: